vpc-sample/tests.cloudbuild.yaml

```diff
@@ -0,0 +1,108 @@
+steps:
+- id: Deploy Restricted Function
+  name: 'gcr.io/cloud-builders/gcloud'
+  dir: '${_SAMPLE_DIR}'
+  args:
+  - functions
```

Comment:
The list format here is confusing to read, and further inconsistent with the Cloud Run syntax. Could we use the 'Deploy to Cloud Run' style for both?

Reply:
Updated - did you mean like this, or did you mean using the bin/bash entrypoint and just writing it all out like the others? 

`  args: ['functions', 'deploy', 'restricted-${SHORT_SHA}', '--runtime=python38', 
        '--trigger-http', '--no-allow-unauthenticated', '--ingress-settings=internal-only',
        '--entry-point=hello_world']`